Jenkinsfile

```diff
@@ -53,7 +77,7 @@ ENVIRONMENT_EOF
     set +o allexport
 
     . ${JABBA_SHELL}
-    jabba use ${JABBA_VERSION}
+    jabba use 1.8
```

Comment:
This op is setting the Java version for what we eventually use to print Java and Maven versions.  Since build version is fixed now at 1.8 I'm not sure we care as much about printing the Java version we use to build things with... although as I think about it we may very well be making _that_ configurable before long to so it really doesn't make sense to take it out either.

But it seems like at a minimum we should print JVM info for the JVM we'll be using for building and for testing, something like:

```
jabba use 1.8 // eventually to be made configurable
echo "Java version for build"
java -version
jabba use $JABBA_VERSION
echo "Java version for tests"
java -version
jabba use 1.8
mvn -v
```

Again, also not a thing we need to fix now... more a marker for future improvements.